README.md

```diff
@@ -111,3 +111,9 @@ You can also join the OpenWhisk Team on Slack [https://openwhisk-team.slack.com]
 The OpenWhisk system is built from a [number of components](docs/dev/modules.md).  The picture below groups the components by their GitHub repos. Please open issues for a component against the appropriate repo (if in doubt just open against the main openwhisk repo).
 
 ![component/repo mapping](docs/images/components_to_repos.png)
+
+### What happens on an invocation?
+
+This diagram depicts the steps which take place within Openwhisk when an action is invoked by the user:
+
+![component/repo mapping](docs/images/Openwhisk-flow.png)
```

Comment:
An ack is also sent via Kafka.

Reply:
![image](https://user-images.githubusercontent.com/11749867/197671285-4cc9c399-a2a9-4ef3-a4d9-c76b6433b84f.png)
Seems `pre-warm container available?` repeated?  if prewarm container is not available, it would do `cold start`

Reply:
Please check now.

Reply:
@Rajiv2605 , i saw you fixed 👍